uploaders/azure.go

```diff
@@ -0,0 +1,85 @@
+// Copyright (c) 2022 Contributors to the Eclipse Foundation
+//
+// See the NOTICE file(s) distributed with this work for additional
+// information regarding copyright ownership.
+//
+// This program and the accompanying materials are made available under the
+// terms of the Eclipse Public License 2.0 which is available at
+// http://www.eclipse.org/legal/epl-2.0
+//
+// SPDX-License-Identifier: EPL-2.0
+
+package uploaders
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/Azure/azure-sdk-for-go/sdk/storage/azblob"
+)
+
+// Constants for Azure upload 'start' operation options
+const (
+	StorageProviderAzure = "azure"
+
+	AzureEndpoint      = "azure.storage.endpoint"
+	AzureSas           = "azure.shared.access.signature"
+	AzureContainerName = "azure.blob.container"
+)
+
+// AzureUploader handles upload to Azure Blob storage
+type AzureUploader struct {
+	endpoint  string
+	sas       string
+	container string
+}
+
+// NewAzureUploader construct new AWSUploader from the provided 'start' operation options
```

Comment:
typo

Reply:
@ttttodorov - could you please resolve the conversation if the typo is fixed?